src/main/scala/com/amazon/deequ/checks/Check.scala

```diff
@@ -303,6 +304,28 @@ case class Check(
     addConstraint(histogramConstraint(column, assertion, binningUdf, maxBins, hint))
   }
 
+  /**
+   * Creates a constraint that asserts on column's sketch size.
+   *
+   * @param column    Column to run the assertion on
+   * @param assertion Function that receives a Distribution input parameter and returns a boolean.
+   *                  E.g
+   *                  .hasLargeKLLSketchSize("att2", _.parameters(1) >= 16,
+   *                  kllParameters = Option(Seq(2, 0.64, 2)))
+   * @param kllParameters parameters of KLL Sketch
+   * @param hint A hint to provide additional context why a constraint could have failed
+   * @return
+   */
+  def hasLargeKLLSketchSize(
```

Comment:
I am a bit unsure about the name. Basically, we get a sketch for the column and can then validate it manually, right?

Reply:
maybe we just call the method `kllSketchSatisfies` ?